## 📋 Quality Gate Results

| Check | Status |
|-------|--------|
| Knip | ✅ success |
| Lint | ❌ failure |
| Build | ✅ success |
| Infra Tests | ❌ skipped |
| Unit Tests | ❌ skipped |
| Component Tests | ❌ skipped |
| Perf Tests | ❌ skipped |
| Visual Tests | ❌ skipped |


### ❌ Lint Failure Details
```text

> hrm@0.30.0 lint /home/runner/work/hrm/hrm
> eslint app/ components/ constants/ context/ hooks/ lib/ services/ tests/ types/ utils/ server.ts proxy.ts --cache


/home/runner/work/hrm/hrm/app/api/spotify/control/route.ts
   21:12  warning  '_e' is defined but never used  unused-imports/no-unused-vars
  123:16  warning  '_e' is defined but never used  unused-imports/no-unused-vars

/home/runner/work/hrm/hrm/hooks/useBluetoothHRM.test.ts
  161:16  warning  '_e' is defined but never used  unused-imports/no-unused-vars

/home/runner/work/hrm/hrm/hooks/useBluetoothHRM.ts
  519:18  warning  '_err' is defined but never used  unused-imports/no-unused-vars

/home/runner/work/hrm/hrm/hooks/usePersistentStorage.ts
  14:12  warning  '_e' is defined but never used  unused-imports/no-unused-vars

/home/runner/work/hrm/hrm/lib/spotify.ts
  56:14  warning  '_e' is defined but never used  unused-imports/no-unused-vars

/home/runner/work/hrm/hrm/lib/utils.ts
  53:50  error  Replace `.toString()` with `⏎······.toString()⏎······`  prettier/prettier

/home/runner/work/hrm/hrm/services/spotifyPolling.ts
  126:13  warning  'refresh_token' is assigned a value but never used. Allowed unused vars must match /^_/u  unused-imports/no-unused-vars

/home/runner/work/hrm/hrm/tests/integration/test-helpers.ts
  70:18  warning  '_e' is defined but never used  unused-imports/no-unused-vars

/home/runner/work/hrm/hrm/tests/playwright/infrastructure.spec.ts
   98:16  warning  '_e' is defined but never used  unused-imports/no-unused-vars
  140:16  warning  '_e' is defined but never used  unused-imports/no-unused-vars

/home/runner/work/hrm/hrm/tests/playwright/lib/visual.ts
  89:12  warning  '_e' is defined but never used  unused-imports/no-unused-vars

/home/runner/work/hrm/hrm/tests/playwright/performance.spec.ts
  57:16  warning  '_error' is defined but never used  unused-imports/no-unused-vars

/home/runner/work/hrm/hrm/tests/playwright/timer-ui-sync.spec.ts
  50:14  warning  '_e' is defined but never used  unused-imports/no-unused-vars

✖ 14 problems (1 error, 13 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

 ELIFECYCLE  Command failed with exit code 1.
```
### ❌ Infrastructure Test Failure Details
```text
Log file not found.
```
### ❌ Unit Test Failure Details
```text
Log file not found.
```
### ❌ Component Test Failure Details
```text
Log file not found.
```
### ❌ Visual Test Failure Details
```text
Log file not found.
```
### ❌ Performance Test Failure Details
```text
Log file not found.
```
⚠️ **Some checks failed.** Full logs available in [workflow artifacts](https://github.com/arii/hrm/actions/runs/21551656661).

---
> Report generated for commit: `20900368a4ef88269f21694d046338a549a31927`

--- a/lib/utils.ts
+++ b/lib/utils.ts
@@ -50,7 +50,9 @@ export const formatDuration = (
     unit === 'milliseconds' ? Math.floor(duration / 1000) : duration
 
   if (format === 'MM:SS') {
-    const minutes = Math.floor(totalSeconds / 60).toString().padStart(2, '0')
+    const minutes = Math.floor(totalSeconds / 60)
+      .toString()
+      .padStart(2, '0')
     const seconds = Math.floor(totalSeconds % 60)
       .toString()
       .padStart(2, '0')

